banzai/utils/fits_utils.py

```diff
@@ -233,3 +237,69 @@ def get_configuration_mode(header):
 
     header['CONFMODE'] = configuration_mode
     return configuration_mode
+
+
+def get_primary_header(file_info, runtime_context):
+    try:
+        hdulist = open_fits_file(file_info, runtime_context)
+        return hdulist[0].header
```

Comment:
Double check that open_fits_file deals with fpacked files correctly and does the rearranging correctly.

Reply:
This does indeed do what we expect.